File: deno/src/cron/cron_job/cron_job.resolver.ts

/**
 * 手动执行定时任务
 */
export async function runCronJob(
  id: CronJobId,
) {
  const {
    runCronJob,
  } = await import("./cron_job.service.ts");
  
  const res = await runCronJob(id);
  return res;
}

File: rust/app/cron/cron_job/cron_job_resolver.rs
use color_eyre::eyre::Result;

use generated::common::context::Options;

use generated::cron::cron_job::cron_job_model::CronJobId;

use super::cron_job_service;

/// 手动执行定时任务
pub async fn run_cron_job(
  id: CronJobId,
  options: Option<Options>,
) -> Result<String> {
  
  let res = cron_job_service::run_cron_job(
    id,
    options,
  ).await?;
  
  Ok(res)
}
